internal.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import "strings"
+
+func IsInternal(pkg, subpkg string) bool {
+	//Skip if any is stdlib
```

Comment:
This comment is missing a space after `//`. Idiomatic Go comments have a space to distinguish them from directives like `//go:generate`, etc. It's also for consistency with other comments.